Source/WebCore/inspector/InspectorStyleSheet.cpp

```diff
@@ -1052,6 +1051,7 @@ void InspectorStyleSheet::reparseStyleSheet(const String& text)
     {
         CSSStyleSheet::RuleMutationScope mutationScope(m_pageStyleSheet.get());
         m_pageStyleSheet->contents().parseString(text);
+        m_pageStyleSheet->contents().transformStyleRuleToNesting();
```

Comment:
Is there ever a time we wouldn't want to do this after parsing sheet text? If not, why wouldn't calling `parseString` put the contents into the correct state itself?

Reply:
Yes indeed, it should be handled somewhere in the callee code